### DISCUSSION TITLE
Questions on Embargo Datasets

### DISCUSSION THREAD

**Discussion by AliyaAbl** [dandi/helpdesk#119] [Q&A]
Hello!

I have three questions regarding restricting access to a portion of my data, intended for training ML models. We aim to store part of the data as a held-out test set, not accessible publicly. Our goal is to centralize our data, including this subset, for potential future publication. However, we don't have a definitive timeline for when this embargo will be lifted.

Q1) Is it possible to set an embargo on a subset of a dandiset?
Q2) Is it possible to set an embargo on a dandiset (or it's subset) and later update its duration?
Q3) Can we embargo a dataset without an NIH affiliation?

Thank you @satra @yarikoptic @aaronkanzer!

**Comment by satra** [comment]
> Q1) Is it possible to set an embargo on a subset of a dandiset?

not really at the moment. you will need to create two dandisets one embargoed, one not.

> Q2) Is it possible to set an embargo on a dandiset (or it's subset) and later update its duration?

according to our current policies it will be similar to NIH.

> Q3) Can we embargo a dataset without an NIH affiliation?

While officially it requests an NIH award, we can figure out a way to do so.

However, i would say a test set is more like a restricted dandiset rather than an embargoed one. We don't have an option for restricted right now, but that will appear in 2024. For the moment you can use embargo to achieve the same policy.

**Reply by AliyaAbl** [reply]
Thank you, Satra! I will create two dandisets then, one embargoed and one not. Regarding Q3: Which steps need to be taken to embargo a dataset without NIH affiliation?

**Reply by satra** [reply]
Just put information about your source of support for the project for now.

**Reply by AliyaAbl** [reply]
Hi Satra, a quick question for clarification: embargoed DandiSets will stay embargoed until I publish them manually - right? I am asking because I cannot find where to provide an anticipated end date for the embargo (I only see the 'unembargo' button).

**Reply by satra** [reply]
correct they will only be unembargoed when pushed manually. we will (in the future) start sending reminders if a dataset remains embargoed for too long. we will also be clarifying elements of the policy in our handbook in the not too distant future to streamline the process. for now, all of the above holds.
